modules/understanding-pod-error-states.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * support/investigating-pod-issues.adoc
+
+[id="understanding-pod-error-states_{context}"]
+= Understanding pod error states
+
+Pod failures return explicit error states which can be observed in the `status` field in the output of `oc get pods`. Pod error states cover image, container and container network related failures.
+
+The following table provides a list of pod error states along with their descriptions.
+
+.Pod error states
+[cols="1,4",options="header"]
+|===
+| Pod error state | Description
+
+2+^s| Images error states
+
+| `ErrImagePull`
+|	Generic image retrieval error.
+
+| `ErrImagePullBackOff`
+| Image retrieval failed and is backed off.
+
+| `ErrInvalidImageName`
+| The specified image name was invalid.
+
+| `ErrImageInspect`
+| Image inspection did not succeed.
+
+| `ErrImageNeverPull`
+| `PullPolicy` is set to `NeverPullImage` and the target image is not present locally on the host.
+
+| `ErrRegistryUnavailable`
+| When attempting to retrieve an image from a registry, an HTTP error was encountered.
+
+2+^s| Container error states
+
+| `ErrContainerNotFound`
+| The specified container is either not present or not managed by kubelet, within the declared pod.
+
+| `ErrRunInitContainer`
+| Container initialization failed.
+
+| `ErrRunContainer`
+| None of the pod's containers started successfully.
+
+| `ErrKillContainer`
+| None of the pod's containers were killed successfully.
+
+| `ErrCrashLoopBackOff`
+| A container has terminated. kubelet will not attempt to restart it.
+
+| `ErrVerifyNonRoot`
+| This error is encountered when container or image runs with root privileges.
```

Comment:
rm "This error is encountered when" to conform with other descriptions. Suggest something like, "A container or image ran with root privileges" or "A container or image tried running with root privileges" -- whichever is more accurate.

Reply:
Thanks Bob. Now resolved.